### PR TITLE
Add dual MIT/Apache2 licenses

### PR DESCRIPTION
**Pull request by jbowen93**
TODO: Is "Union Labs Inc." the appropriate entity?

### CODE REVIEW

File: LICENSE-MIT [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Union Labs Inc.
```
Comment (KaiserKarel) [inline LICENSE-MIT:3]:
```suggestion
Copyright (c) 2024 Union.fi Labs, Inc.
```

Comment (KaiserKarel) [reply LICENSE-MIT:3]: Union Labs was taken :(

File: LICENSE-APACHE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2024 Union Labs Inc.
```
Comment (KaiserKarel) [inline LICENSE-APACHE:189]:
```suggestion
   Copyright 2024 Union.fi Labs, Inc.
```

**Comment by benluelo** [comment]
thanks for the PR! we follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for all commits, so please edit your commits and force push to the branch

**Review by aeryz — APPROVED** [review]

**Comment by aeryz** [comment]
Hey @jbowen93, thanks for the PR. We require all commits to be signed. Can you do that please :)

**Comment by ConejoCapital** [comment]
lgtm 🥹🫶❤️

**Comment by jbowen93** [comment]
I think I got it all figured out

**Pull request merged**